crates/wasmprinter/Cargo.toml

```diff
@@ -25,5 +25,6 @@ termcolor = { workspace = true }
 wat = { path = "../wat" }
 
 [features]
-default = ['component-model']
+default = ['component-model', 'simd']
 component-model = ['wasmparser/component-model']
+simd = ['wasmparser/simd']
```

Comment:
Similar to wasm-encoder, what do you think about unconditionally enabling simd here? That would avoid the need for more `#[cfg]` and I don't think there's as compelling of a use case to making this conditional on this crate as there is for wasmparser